src/view/actions/resetEventMergeId.jsx

```diff
@@ -36,10 +37,9 @@ const getSettings = ({ values }) => {
 };
 
 const validationSchema = object().shape({
-  eventMergeId: string().matches(
```

Comment:
A value of an empty string wouldn't pass this `matches` test, but a valued of `undefined` will. Since `eventMergeId` is a required field, we need to use `.required` to ensure `undefined` fails validation.